parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -261,7 +261,9 @@ async def entry_rename(
             await self.local_storage.set_manifest(parent.id, new_parent)
 
         # Send event
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id)
+        self._send_event(
+            CoreEvent.FS_ENTRY_UPDATED, id=parent.id, reason=FSEntryUpdatedReason.ENTRY_RENAME
```

Comment:
I think we need to provide more informations about what is renamed here (typically `entry_id`, `entry_source_name` and `entry_destination_name`)